sql/qc_geospatial_check.sql

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS geospatial_check;
+CREATE TABLE IF NOT EXISTS geospatial_check(
+    result character varying
+);
+
+INSERT INTO geospatial_check (
+select  jsonb_agg(t) as result
+from (
+    select jsonb_agg(json_build_object('uid', tmp.uid, 'hnum', tmp.HNUM, 'sname', tmp.sname, 
+    'adress', tmp.address, 'parcelname', tmp.PARCELNAME, 'agency', tmp.AGENCY, 
+    'latitude', tmp.latitude,'longitude', tmp.longitude, 'v', tmp.v)) as values, 
+                              'projects_not_within_NYC' as field
+	from (SELECT a.uid, a.HNUM, a.sname, a.address, a.PARCELNAME, a.AGENCY, a.latitude, a.longitude, a.v
+          FROM dcp_colp a, 
+               (SELECT ST_Union(wkb_geometry) geom
+               FROM dcp_boroboundaries_wi) combined 
```

Comment:
Okay so we load in the `dcp_boroboundaries_wi` in the `dataloading` for the geospatial checks